src/gameplay/game.ts

```diff
@@ -1970,6 +1974,16 @@ class Game extends Room {
       return;
     }
 
+    if (this.gameTimer.dateTimerExpires.getTime() === new Date(0).getTime()) {
```

Comment:
Hmmm, can we instead have a method on GameTimer called `IsTimerSet()` or something and inside it does this check?